Emby.Drawing/ImageProcessor.cs

```diff
@@ -126,21 +122,9 @@ public async Task<(string path, string mimeType, DateTime dateModified)> Process
                 throw new ArgumentNullException(nameof(options));
             }
 
-            var libraryManager = _libraryManager();
-
             ItemImageInfo originalImage = options.Image;
             BaseItem item = options.Item;
 
-            if (!originalImage.IsLocalFile)
-            {
-                if (item == null)
-                {
-                    item = libraryManager.GetItemById(options.ItemId);
-                }
-
-                originalImage = await libraryManager.ConvertImageToLocal(item, originalImage, options.ImageIndex).ConfigureAwait(false);
-            }
```

Comment:
This logic was moved [here](https://github.com/jellyfin/jellyfin/pull/2765/files#diff-94d9c9c6f3144cc37f203fc619673b63R610) in order to eliminate the dependency on `libraryManager` in this class